CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/utilities/DataAccessUtilities.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.dataaccess.utilities;
+
+import au.gov.asd.tac.constellation.graph.Graph;
+import au.gov.asd.tac.constellation.graph.ReadableGraph;
+import au.gov.asd.tac.constellation.graph.WritableGraph;
+import au.gov.asd.tac.constellation.plugins.parameters.PluginParameter;
+import au.gov.asd.tac.constellation.views.dataaccess.panes.DataAccessPane;
+import au.gov.asd.tac.constellation.views.dataaccess.DataAccessViewTopComponent;
+import au.gov.asd.tac.constellation.views.dataaccess.components.DataAccessTabPane;
+import au.gov.asd.tac.constellation.views.dataaccess.state.DataAccessConcept;
+import au.gov.asd.tac.constellation.views.dataaccess.state.DataAccessState;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Map;
+import javafx.scene.control.Tab;
+import javafx.scene.control.TabPane;
+import javax.swing.SwingUtilities;
+import org.openide.util.Exceptions;
+import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
+
+/**
+ *
+ * @author algol
+ */
+public class DataAccessUtilities {
+
+    /**
+     * Private constructor to prevent external initialization.
+     */
+    private DataAccessUtilities() {
+        
+    }
+    
+    /**
+     * A convenience method for getting the Pane used by the Data Access view.
+     * <p>
+     * This allows dialog boxes to be given parents, for example.
+     * <p>
+     * If the Data Access view is not opened, it will be.
+     *
+     * @return The Pane used by the Data Access view.
+     */
+    public static DataAccessPane getDataAccessPane() {
+        if (SwingUtilities.isEventDispatchThread()) {
+            return getInternalDataAccessPane();
+        }
+
+        final DataAccessPane[] panes = new DataAccessPane[1];
+        try {
+            SwingUtilities.invokeAndWait(() -> panes[0] = getInternalDataAccessPane());
+        } catch (InterruptedException ex) {
+            Exceptions.printStackTrace(ex);
+            Thread.currentThread().interrupt();
+        } catch (InvocationTargetException ex) {
+            Exceptions.printStackTrace(ex);
```

Comment:
exceptions final. Also if you could replace the stack trace prints with logs, that would be great